packages/react-examples/src/react/SwatchColorPicker/SwatchColorPicker.Basic.Example.tsx

```diff
@@ -32,19 +32,30 @@ export const SwatchColorPickerBasicExample: React.FunctionComponent = () => {
 
   return (
     <div>
-      <div>Simple circle swatch color picker:</div>
-      <SwatchColorPicker columnCount={5} cellShape={'circle'} colorCells={colorCellsExample1} />
-      <div>Simple square swatch color picker with default size of 20px:</div>
-      <SwatchColorPicker columnCount={5} cellShape={'square'} colorCells={colorCellsExample1} />
-      <div>Simple square swatch color picker with custom size of 35px:</div>
+      <div id="circleLabel">Simple circle swatch color picker:</div>
+      <SwatchColorPicker
+        columnCount={5}
+        cellShape={'circle'}
+        colorCells={colorCellsExample1}
+        aria-labelledby="circleLabel"
```

Comment:
Can you use `useId` to generate the IDs? That's a good idea with examples to prevent issues if someone copy-pastes this (either into another example or possibly their app).